test/integration/lookout_controller_e2e_test.go

```diff
@@ -30,27 +40,155 @@ spec:
         operator: in
 `
 
-var _ = Describe("Armada Operator", func() {
-	When("User applies Lookout YAML using kubectl", func() {
-		It("Kubernetes should create Lookout Kubernetes resources", func() {
+var lookoutYaml2 = `apiVersion: install.armadaproject.io/v1alpha1
```

Comment:
Could you refactor these tests to use the resource files rather than the inline yaml?

Reply:
Yeah I will need to resolve the merge conflicts anyways